File: extension/core/src/main/java/io/holunda/camunda/bpm/data/writer/VariableWriter.java
package io.holunda.camunda.bpm.data.writer;

import io.holunda.camunda.bpm.data.adapter.WriteAdapter;

/**
 * Inverting calls to {@link WriteAdapter}.
 *
 * @param <S> type of concrete Writer for fluent usage.
 */
public interface VariableWriter<S extends VariableWriter<S>> extends LocalVariableWriter<S>, GlobalVariableWriter<S> {

}

File: extension/core/src/main/kotlin/io/holunda/camunda/bpm/data/writer/VariableWriter.kt
package io.holunda.camunda.bpm.data.writer

/**
 * Inverting calls to [io.holunda.camunda.bpm.data.adapter.WriteAdapter].
 *
 * @param <S> type of concrete Writer for fluent usage.
</S> */
interface VariableWriter<S : VariableWriter<S>> : LocalVariableWriter<S>, GlobalVariableWriter<S>
